src/downshift.js

```diff
@@ -478,13 +480,32 @@ class Downshift extends Component {
     {refKey = 'ref', ...rest} = {},
     {suppressRefError = false} = {},
   ) => {
+    const {isOpen} = this.getState()
     // this is used in the render to know whether the user has called getRootProps.
     // It uses that to know whether to apply the props automatically
     this.getRootProps.called = true
     this.getRootProps.refKey = refKey
     this.getRootProps.suppressRefError = suppressRefError
+
+    // TODO: Error Handling?
```

Comment:
Yes, let's do that :+1: Similar to how the label's `htmlFor` and `inputId` are validated. Except instead of throwing an error, let's use `console.error` and wrap it all in `process.env.NODE_ENV === 'production'` so it can be eliminated for production. In a future breaking version of `downshift`, I want to move all the validation to this to make downshift smaller.